src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
 body {
+  display: flex;
   margin: 0;
+  background-color: #e5e5e5;
 }
+
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/star';
```

Comment:
It's good that you're using SCSS partials to organize your code. However, it's better to import utility files like `variables` at the beginning of your main SCSS file. This way, the variables can be accessed by any subsequent styles. So the import of `variables` should be above all other imports.